src/main/java/it/smartcommunitylab/aac/webauthn/controller/WebAuthnRegistrationController.java

```diff
@@ -62,7 +71,7 @@
     @RequestMapping(value = "/auth/webauthn/register/{providerId}", method = RequestMethod.GET)
     public String registrationPage(@PathVariable("providerId") String providerId) {
         try {
-            final boolean canRegister = webAuthnRpServiceReigistrationRepository.getProviderConfig(providerId)
+            final boolean canRegister = registrationRepository.findByProviderId(providerId)
```

Comment:
repository find could return null, you need to check.

Reply:
Added the check with a04ed36b186db47a3d3a9aa1da66a6f7a1611506